docs/root/intro/version_history.rst

```diff
@@ -23,6 +23,7 @@ Version history
 * http: added :ref:`max request headers size <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.max_request_headers_size_kb>`. The default behaviour is unchanged.
 * redis: added :ref:`success and error stats <config_network_filters_redis_proxy_per_command_stats>` for commands.
 * redis: added :ref:`latency stats <config_network_filters_redis_proxy_per_command_stats>` for commands.
+* redis: migrate from std::hash to `murmurHash2 <https://sites.google.com/site/murmurhash>`_.
```

Comment:
Can you be more specific about what is happening here? Basically that for hash ring selection we are migrating from `std::hash` to an equivalent for most all production uses, etc. See the other release note from @HenryYYang and maybe use that for inspiration?